lib/internal/test_runner/test.js

```diff
@@ -552,7 +552,8 @@ class Test extends AsyncResource {
         file: loc[2],
       };
 
-      if (this.config.sourceMaps === true) {
+      // TODO(RedYetiDev): This should be supported with code coverage
+      if (this.config.sourceMaps && !this.config.coverage) {
```

Comment:
**Review Note** When Code Coverage is enabled, the source map is loaded in the coverage constructor, and this statement messes up the lines. Not quite sure why, but skipping it has no negative impact.